src/CodeFixes/VerifyOverridableMembersFixer.cs

```diff
@@ -0,0 +1,138 @@
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Moq.Analyzers;
+
+/// <summary>
+/// Fixes Moq.Verify calls on non-overridable members by making them virtual.
+/// </summary>
+[ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(VerifyOverridableMembersFixer))]
+[Shared]
+public sealed class VerifyOverridableMembersFixer : CodeFixProvider
+{
+    /// <inheritdoc/>
+    public override ImmutableArray<string> FixableDiagnosticIds { get; } = ImmutableArray.Create(DiagnosticIds.VerifyOnlyUsedForOverridableMembers);
+
+    /// <inheritdoc/>
+    public override FixAllProvider GetFixAllProvider() => WellKnownFixAllProviders.BatchFixer;
+
+    /// <inheritdoc/>
+    public override async Task RegisterCodeFixesAsync(CodeFixContext context)
+    {
+        SyntaxNode? root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
+        if (root == null)
+        {
+            return;
+        }
+
+        Diagnostic diagnostic = context.Diagnostics.First();
+        Microsoft.CodeAnalysis.Text.TextSpan diagnosticSpan = diagnostic.Location.SourceSpan;
+
+        SyntaxNode? invocationNode = root.FindNode(diagnosticSpan, getInnermostNodeForTie: true);
+        if (invocationNode is not InvocationExpressionSyntax)
+        {
+            invocationNode = invocationNode.FirstAncestorOrSelf<InvocationExpressionSyntax>();
+        }
+
+        if (invocationNode == null)
+        {
+            return;
+        }
+
+        SemanticModel? semanticModel = await context.Document.GetSemanticModelAsync(context.CancellationToken).ConfigureAwait(false);
+        if (semanticModel == null)
+        {
+            return;
+        }
+
+        IOperation? operation = semanticModel.GetOperation(invocationNode, context.CancellationToken);
+        if (operation is not IInvocationOperation invocationOperation)
+        {
+            return;
+        }
+
+        ISymbol? mockedMemberSymbol = TryGetMockedMemberSymbol(invocationOperation);
+        if (mockedMemberSymbol == null)
+        {
+            return;
+        }
+
+        // Check if we should offer a fix.
+        if (mockedMemberSymbol is not IPropertySymbol and not IMethodSymbol ||
+            mockedMemberSymbol.IsSealed ||
+            mockedMemberSymbol.IsAbstract ||
+            mockedMemberSymbol.IsVirtual ||
+            mockedMemberSymbol.ContainingType.TypeKind == TypeKind.Interface)
+        {
+            return;
+        }
+
+        context.RegisterCodeFix(
+            CodeAction.Create(
+                "Make member virtual",
+                cancellationToken => MakeMemberVirtualAsync(context.Document, mockedMemberSymbol, cancellationToken),
+                nameof(VerifyOverridableMembersFixer)),
+            diagnostic);
+    }
+
+    private static ISymbol? TryGetMockedMemberSymbol(IInvocationOperation moqVerifyInvocation)
+    {
+        if (moqVerifyInvocation.Arguments.Length == 0)
+        {
+            return null;
```

Comment:
This line is uncovered. Write a unit test for this scenario if it is valid; otherwise, refactor so we don't have uncovered lines.